### ISSUE TITLE
Manual trigger backup

### ISSUE THREAD

**Issue by N1c093** [thomasmauerer/hassio-addons#15]
Hello,
first of all, thank you for this great addon.

Is it currently possible to manual trigger an backup? I set the schedule inside the addon so it creates backup automatically.
```yaml
trigger_time: '03:00'
trigger_days:
  - Mon
  - Wed
  - Fri
  - Sun
```

I think the following service call gets disabled when you set a schedule inside the add on. Is it right?
```yaml
service: hassio.addon_stdin
data:
  addon: 15d21743_samba_backup
  input: trigger
```

It would be great to still have the possibility to create manual backups. For example before you install a new version of HA.

Thanks in advance.

**Comment by niro1987** [comment]
you can add ad many triggers as you'de like to an automation.

Set `trigger_time` to `manual` and add the following automation to run daily backups. This way you can run the automation manually or even post `trigger` to mqtt topic `samba_backup/status` for a manual backup (with a button card for example).


**automation.yaml**
```yaml
- id: sambabackup
  alias: Samba Backup
  description: ''
  trigger:
  - payload: trigger
    platform: mqtt
    topic: samba_backup/status
  - at: 00:00:00
    platform: time
  condition: []
  action:
  - data:
      addon: 15d21743_samba_backup
      input: trigger
    service: hassio.addon_stdin
```

**Comment by N1c093** [comment]
Thanks for your reply. 

But I like to use the build in schedule of the add on and no extra automation. I just like to have the possibility to create a manual backup. I think this is currently not possible

**Comment by thomasmauerer** [comment]
Hi. That's currently not possible. The only way to have multiple triggers at the same time is by using a HA automation as described above by Niels.

Unfortunately, this would introduce some difficulties and problems (multiple processes in docker container, synchronization, etc.)

**Comment by thomasmauerer** [comment]
Ok I'm working on it. Will be released soon.

**Comment by brucek1642** [comment]
Here's how I do a manual backup. I also had a automation schedule before migrating to this add-on so it made sense for me to keep the scheduling external. Basically I call my main automation the one that contains the schedule with a button.  I use the custom-button card but does work with regular button card. You should also be able to trigger the automation from another script or automation. Or even simpler go to the automation UI and click on execute.

In the configuration

trigger_time: manual

The automation 

- id: '1591061422068'
  alias: Auto Samba Backup and delayed host reboot
  description: ''
  trigger:
  - at: 03:00:00
    platform: time
  condition:
  - condition: time
    weekday:
    - mon
    - wed
    - fri
    - sun
  action:
  - data:
      addon: 15d21743_samba_backup
      input: trigger
    service: hassio.addon_stdin
  - data: {}
    service: script.delayed_reboot1588260801473

Here's my lovelace code

color: default
entity: automation.auto_samba_backup_and_delayed_host_reboot
icon: 'mdi:content-save-all'
icon_name_state: null
layout: icon_name
name: Trigger Backup
tap_action:
  action: call-service
  service: automation.trigger
  service_data:
    entity_id: automation.auto_samba_backup_and_delayed_host_reboot
type: 'custom:button-card'

Footnote to Thomas. 

Thanks for your efforts. The code has been running very reliable and making my life easier.

**Comment by thomasmauerer** [comment]
Implemented in v3.0.

@brucek1642  Your automation will still work, but you can now also use the time-based schedule directly from the add-on and still trigger a manual backup with a button-card. Thanks for your nice feedback!
